README.md

```diff
@@ -40,13 +40,15 @@ sure that we're all on the same page.
   done the work _on theirs directly_. [Here][pr] is a great example of a pull
   request on the `Ruby` codebase.
 
-Let's go over a conceptual, hypothetical example. It's OK if this feels a bit confusing at
-first. You'll work through this countless times and eventually your brain and
-fingers will both grasp what's going on. Let's look at the following example:
+We're going to go go over a conceptual, hypothetical example. THIS IS ONLY AN 
+EXAMPLE. It's OK if this feels a bit confusing at first. You'll work through 
+this countless times and eventually your brain and fingers will both grasp 
+what's going on. Let's look at the following scenario:
 
 ### Pull Request to a Source Repository
 
-1. Let's pretend that the learn-co-students organization has a repo called awesome-lab, and we make a "fork" from this repo at `https://github.com/learn-co-students/awesome-lab`.
+1. Let's pretend that the learn-co-students organization has a repo called awesome-lab, and 
+   we make a "fork" of the repo `https://github.com/learn-co-students/awesome-lab`.
```

Comment:
"..again. Do not actually do this, the point is to build the _concept_ of what forking does and we need a name "awesome-lab.""

some sort of coaching closer to the point where the error occurs.